docs/workflows/common-workflow-steps/sauce-labs-saucectl.md

```diff
@@ -22,25 +22,15 @@ Below are the workflow steps required before running the **Saucectl** step, list
 | --------------------------- | ----------------------------------------------------------------------------- |
 | [Android Build for UI Testing](/workflows/android-specific-workflow-steps/android-build-for-ui-testing) | This step generates the required Android test application outputs needed for testing on Sauce Labs. |
```

Comment:
We should make "Android Build for UI Testing" bold in order to align the formatting with the iOS section.

Reply:
@boztopuz Any updates for this item?

Reply:
@csonuryilmaz I fixed it last commit